c2corg_ui/static/js/lang.js

```diff
@@ -84,6 +84,7 @@ app.LangController.prototype.updateCulture = function() {
   this.gettextCatalog_.setCurrentLanguage(this.culture);
   this.gettextCatalog_.loadRemote(
       this.langUrlTemplate_.replace('__lang__', this.culture));
+  window.localStorage.setItem('interface_lang', this.culture);
```

Comment:
You think that window.localStorage might not be available and would cause an error?
By the way, I considered using sessionStorage as you did for the auth but thought  that it was a bit overkill since it's not  a sensitive info.